app.js

```diff
@@ -0,0 +1,20 @@
+let express = require('express');
```

Comment:
Consider changing such import variables to "const".
Let's follow general guidelines about the meaning and usage of let and const.
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

In short, consider defaulting to "const" at all times, if the variable is not going to change.